pkg/rancher-desktop/main/deploymentProfiles.ts

```diff
@@ -13,7 +13,10 @@ import { RecursivePartial } from '@pkg/utils/typeUtils';
 
 const console = Logging.deploymentProfile;
 
-const REGISTRY_PATH_PROFILE = ['SOFTWARE', 'Rancher Desktop', 'Profile'];
+const REGISTRY_PATH_PROFILE = [
```

Comment:
This should be `REGISTRY_PATH_PROFILES` because there's now more than one.

Reply:
Changed to REGISTRY_PROFILES_PATH.